pkgs/development/ocaml-modules/postgresql/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildDunePackage, postgresql }:
+
+buildDunePackage rec {
+  pname = "postgresql";
+  version = "4.6.3";
+
+  minimumOCamlVersion = "4.08";
+
+  src = fetchFromGitHub {
+    owner = "mmottl";
+    repo = "postgresql-ocaml";
+    rev = version;
+    sha256 = "0fd96qqwkwjhv6pawk4wivwncszkif0sq05f0g5gd28jzwrsvpqr";
+  };
+
+  buildInputs = [ postgresql ];
+
+  meta = {
+    description = "Bindings to the PostgreSQL library";
+    license = "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception";
```

Comment:
Many OCaml derivations in `nixpkgs` just use `lib.licenses.lgpl21Plus` and do not worry about the linking exception.